controller-server/src/main/java/com/yahoo/vespa/hosted/controller/Controller.java

```diff
@@ -78,28 +78,26 @@
      */
     @Inject
     public Controller(CuratorDb curator, RotationsConfig rotationsConfig,
-                      ZoneRegistry zoneRegistry,
```

Comment:
`ZoneRegistryMock` can be removed from `ControllerContainerTest` now.

Reply:
Indeed.